src/agent0/chainsync/dashboard/build_wallet_positions.py

```diff
@@ -0,0 +1,115 @@
+"""Builds the position dataframe for the dashboard."""
+
+import pandas as pd
+
+from .usernames import abbreviate_address, map_addresses
+
+
+def build_wallet_positions(
+    positions_snapshot: pd.DataFrame,
+    user_map: pd.DataFrame,
+    hyperdrive_addr_map: pd.DataFrame,
+) -> pd.DataFrame:
+    """Show recent trades wrt a wallet.
```

Comment:
`dashboard/build_ticker.py::build_wallet_ticker()` has the same description as this fn `dashboard/build_wallet_positions.py::build_wallet_positions()`. 

> Show recent trades wrt a wallet.

can you update one/both to so that the difference between them is clear?